neuralhydrology/modelzoo/transformer.py

```diff
@@ -167,17 +173,32 @@ def __init__(self, embedding_dim, dropout, max_len=5000):
         pe = pe[:,:embedding_dim].unsqueeze(0).transpose(0, 1)
         self.register_buffer('pe', pe)
 
-    def forward(self, x, pos_type):
-        """
+        if position_type.lower() == 'concatenate':
+            self.concatenate = True
+        elif position_type.lower() == 'sum':
+            self.concatenate = False
+        else:
+            raise RuntimeError(f"Unrecognized positional encoding type: {pos_type}")
+
+    def forward(self, x):
+        """Forward pass for positional encoding. Either concatenates or adds the positional encoding to encoder input data.
+
+        Parameters
+        ----------
+        x : torch.Tensor
+            Dimension is sequence length x embedding output dimension. 
```

Comment:
I'd suggest to be a bit more explicit here by including the batch size. Also, I feel like the bracket notation is less ambiguous since `x` is already a variable name.
```suggestion
            Dimension is ``[sequence length, batch size, embedding output dimension]``.
```